docs/features/details/Details.md

```diff
@@ -17,8 +17,6 @@ class UseCase
 class Scenario
```

Comment:
Add the _details_ attribute to the _Branch, Build, UseCase, Scenario, Step_ classes

Reply:
the dependency line is enough here